src/nxt_conf_validation.c

```diff
@@ -1683,14 +1684,9 @@ nxt_conf_vldt_listener(nxt_conf_validation_t *vldt, nxt_str_t *name,
     nxt_conf_value_t *value)
 {
     nxt_int_t       ret;
-    nxt_str_t       str;
     nxt_sockaddr_t  *sa;
 
-    if (nxt_slow_path(nxt_str_dup(vldt->pool, &str, name) == NULL)) {
-        return NXT_ERROR;
-    }
-
-    sa = nxt_sockaddr_parse(vldt->pool, &str);
+    sa = nxt_sockaddr_parse(vldt->pool, name);
```

Comment:
I take it we don't need to strdup it here as this is called from nxt_conf_vldt_object_iterator() where it's already duped?

Reply:
Right.

(rant)
Why can't github show it as a hunk so that I know in which function this is happening?
This is
```diff
@@ -1683,14 +1684,9 @@ nxt_conf_vldt_listener(nxt_conf_validation_t *vldt, nxt_str_t *nam
e,
```
in git
(/rant)